Makefile.am

```diff
@@ -272,13 +272,18 @@ test_unit_test_tpm2_util_SOURCES   = test/unit/test_tpm2_util.c
 abs_build_dir=$(abspath $(builddir))
 
 AM_TESTS_ENVIRONMENT =	\
-	TPM2_ABRMD="tpm2-abrmd" \
-	TPM2_SIM="tpm_server" \
-	PATH=$(abs_build_dir)/tools:$(abs_build_dir)/tools/misc:$(abs_top_srcdir)/test/integration:$(PATH) \
-	TPM2_TOOLS_TEST_FIXTURES=$(abs_top_srcdir)/test/integration/fixtures \
-	dbus-run-session
+	TPM2_ABRMD=tpm2-abrmd; export TPM2_ABRMD; \
+	TPM2_SIM=tpm_server; export TPM2_SIM; \
+	PATH=$(abs_build_dir)/tools:$(abs_build_dir)/tools/misc:$(abs_top_srcdir)/test/integration:$(PATH); \
+	TPM2_TOOLS_TEST_FIXTURES=$(abs_top_srcdir)/test/integration/fixtures; \
+    export TPM2_TOOLS_TEST_FIXTURES;
```

Comment:
indentation. Do you need this export? I don't think you do, but I could be wrong.

Reply:
`TEST_FIXTURES` is used in `createek.sh` and `send.sh`, both fail if we don't export it. We could fix them so that we don't need to export this variable?